locales/en-US/app.ftl

```diff
@@ -1,59 +1,72 @@
 siteName = Firefox Test Pilot
 
+# Page titles, put in the <title> HTML tag.
+[pageTitle]
```

Comment:
Hah, good catch. I was building it like an [ini file](https://en.wikipedia.org/wiki/INI_file#Format), which is very similar.